src/open_inwoner/conf/base.py

```diff
@@ -631,6 +631,9 @@
 AXES_USE_USER_AGENT = True  # Default: False
 AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP = True  # Default: False
 AXES_BEHIND_REVERSE_PROXY = IS_HTTPS
+# By default, Axes obfuscates values for formfields named "password", but the admin
+# interface login formfield name is "auth-password", so we obfuscate that as well
+AXES_PASSWORD_FORM_FIELD = "auth-password"  # nosec
```

Comment:
Volgens mij voorkom je hiermee dat 'password' wordt geobfuscate, toch?
Aangezien een gemeente inloggen met gebruikersnaam+wachtwoord kan gebruiken lijkt het opnemen van auth-password via AXES_SENSITIVE_PARAMETERS dan verstandiger.

Dit graag ook in default-project en OpenFormulieren doorvoeren

Reply:
het komt op hetzelfde neer, de lijst van parameters die gecleared worden begint altijd met "password", daar worden AXES_SENSITIVE_PARAMETERS aan toegevoegd en AXES_PASSWORD_FORM_FIELD ook https://github.com/jazzband/django-axes/blob/448d0cea1a6985dce7ae71b5613d544d5bda401d/axes/helpers.py#L394

Ik kan voor de duidelijkheid anders wel gewoon `AXES_SENSITIVE_PARAMETERS = ["password", "auth-password"]` opnemen, dan is het iig expliciet

Reply:
Ah duidelijk, stond niet vermeld in de docs maar dat is met axes wel vaker het geval. Bij deze akkoord